packages/utils/src/validate/schema.ts

```
@@ -13,8 +13,29 @@
 
-function Schema(descriptor, translate?) {
-  Schema.getSystemMessage = () => Schema.getDefaultMessage(translate)
+// 添加Schema的接口定义
+interface SchemaType {
+  rules: Record<string, any> | null
+  _messages: Record<string, any>
+  define(descriptor: Record<string, any>): void
+  messages(messages?: Record<string, any>): Record<string, any>
+  getSeries(options: any, source: any, source_: any): Record<string, any>
+  mergeMessage(options: any): void
+  validate(source_: any, o?: any, oc?: Function): Promise<any>
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)